pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1179,6 +1179,10 @@ self: super:
   # https://github.com/xu-hao/namespace/issues/1
   namespace = doJailbreak super.namespace;
 
+  # yaml doesn't build its executables (json2yaml, yaml2json) by default:
+  # https://github.com/snoyberg/yaml/issues/194
+  yaml = addBuildDepend self.optparse-applicative (disableCabalFlag "no-exe" super.yaml);
```

Comment:
Can you also add `enableSeparateBinOutput`?

Reply:
OK, you've convinced me this should go in `configuration-nix.nix` since that's where all the other `enableSeparateBinOutput`s are :)